assets/materials/shaders/PBR.hlsl

```diff
@@ -116,108 +112,62 @@ float4 psmain(VSOutput input) : SV_TARGET
         // Normalize the transformed normal
         N = normalize(N);
     }
-    
-    float3 P     = input.positionWS;
-    float3 E     = normalize(Scene.eyePosition - P);
-    float3 R     = reflect(-V, N);
-    float3 Lo    = E; // Outgoing light direction
-    float  cosLo = saturate(dot(N, Lo));
-
-    // The value for F0 is passed in from the application. If in doubt use 0.04 for 
-    // dieletric materials and scale beween 0.04 and the albedo based on the metalness
-    // value.
-    //
-    float3 F0 = Material.F0;
-    F0        = lerp(F0, albedo, metalness);
+
+    // Remap
+    float3 diffuseColor = (1.0 - metalness) * albedo;
+    roughness = roughness * roughness;
+
+    // Calculate F0
+    float  specularReflectance = 0.5;
+    float3 F0 = 0.16 * specularReflectance * specularReflectance * (1 - metalness) + albedo * metalness;    
+
+    // Reflection and cosine angle bewteen N and V 
+    float3 R = reflect(-V, N);
+    float  NoV = saturate(dot(N, V));
 
     // Calculate direct lighting
-    float3 directLighting = (float3)0;
-    for (uint i = 0; i < Scene.lightCount; ++i) {
-        float3 Li   = normalize(Lights[i].position - P); // Incoming light direction
-        float  Lrad = Lights[i].intensity;               // Light radiance
-        float3 Lh   = normalize(Li + Lo);                // Half-vector between Li and Lo
-        float  cosLi = saturate(dot(N, Li));
-        float  cosLh = saturate(dot(N, Lh));
-
-        // Calculate Fresnel term for direct lighting 
-        float3 F = FresnelSchlick(F0, saturate(dot(Lh, Lo)));
-        // Calculate normal distribution for specular BRDF
-        float  D = DistributionGGX(cosLh, roughness);
-        // Calculate geometric attenuation for specular BRDF
-        float  G = GASmithSchlickGGX(cosLi, cosLo, roughness);
-
-        // Diffuse scattering happens due to light being refracted multiple times by a dielectric medium.
-        // Metals on the other hand either reflect or absorb energy, so diffuse contribution is always zero.
-        // To be energy conserving we must scale diffuse BRDF contribution based on Fresnel factor & metalness.
-        //
-        float3 kD = lerp(float3(1, 1, 1) - F, float3(0, 0, 0), metalness);
-
-        // clang-format off
-        // 
-        // Lambert diffuse BRDF
-        // 
-        // We don't scale by 1/PI for lighting & material units to be more convenient.
-        //   See: https://seblagarde.wordpress.com/2012/01/08/pi-or-not-to-pi-in-game-lighting-equation/
-        //
-        // clang-format on
-        float3 diffuseBRDF = kD * albedo;
-
-        // Calculate specular BRDF
-        float3 specularBRDF = (F * D * G) / max(0.00001, 4.0 * cosLi * cosLo);
-    
-        // Accumulate light's total contribution
-        directLighting += (diffuseBRDF + specularBRDF) * Lrad * cosLi;
+    float3 directLighting = 0;
+	for (uint i = 0; i < Scene.lightCount; ++i) {    
```

Comment:
Do we need to fix indentation here?

Reply:
Line 130 here, if it should have one fewer \t, same as line 129

Reply:
Thanks. Fixed.